src/ts/createTyperighterPlugin.ts

```diff
@@ -99,6 +102,13 @@ export interface IPluginOptions<
    * Called when a match decoration is clicked.
    */
   onMatchDecorationClicked?: (match: TMatch) => void;
+
+  /**
```

Comment:
Think this comment may not apply.

Reply:
Whoops - yes I think that probably applied to `enableDevMode`